ui/src/app/subnets/views/FormActions/components/AddVlan.tsx

```diff
@@ -0,0 +1,126 @@
+import { useEffect } from "react";
+
+import { Row, Col, Input } from "@canonical/react-components";
+import { useDispatch, useSelector } from "react-redux";
+import * as Yup from "yup";
+
+import type { FormActionProps } from "../FormActions";
+
+import FabricSelect from "app/base/components/FabricSelect";
+import FormikField from "app/base/components/FormikField";
+import FormikForm from "app/base/components/FormikForm";
+import SpaceSelect from "app/base/components/SpaceSelect";
+import { actions as fabricActions } from "app/store/fabric";
+import fabricSelectors from "app/store/fabric/selectors";
+import { actions as spaceActions } from "app/store/space";
+import spaceSelectors from "app/store/space/selectors";
+import { actions as vlanActions } from "app/store/vlan";
+import vlanSelectors from "app/store/vlan/selectors";
+import { toFormikNumber } from "app/utils";
+
+type AddVlanValues = {
+  vid?: string;
+  name?: string;
+  fabric?: string;
+  space?: string;
```

Comment:
These must actually all be defined since `""` is still a value

Reply:
True, defined these before I've set all defaults to `""`